lib/matplotlib/gridspec.py

```diff
@@ -170,6 +172,68 @@ def _normalize(key, size):  # Includes last index.
 
         return SubplotSpec(self, num1, num2)
 
+    def legend_outside(self, handles=None, labels=None, axs=None,
+                       align='horizontal', **kwargs):
+        """
+        legend for this gridspec, offset from all the subplots.
+
+        See `.Figure.legend_outside` for details on how to call.
```

Comment:
"the *outside* argument to `Figure.legend`"
Dunno whether you want to rename that function now that Figure.legend_outside was renamed, but perhaps not.

Reply:
Ooops, no this definitely still needs some documentation work.  Also the above new stuff is a bit of a mess still, so let me mark WIP.

Reply:
Fixed....